extensions/src/AWSSDK.Extensions.Logging.ILoggerAdaptor/AWSSDK.Extensions.Logging.ILoggerAdaptor.nuspec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package> 
+  <metadata> 
+    <id>AWSSDK.Extensions.Logging.ILoggerAdaptor</id>
+    <title>AWSSDK - Adaptor for Microsoft.Extensions.Logger</title>
```

Comment:
I thought the Microsoft package name was `Microsoft.Extensions.Logging`. Is this name correct? (same comment applies to the description on line 8)

Reply:
Good catch. Fixed in latest commit.